src/seeds/users/001_users.ts

```diff
@@ -0,0 +1,82 @@
+import * as bcrypt from 'bcrypt';
+import { randomUUID } from 'crypto';
+import { Knex } from 'knex';
+
+const initialUsers = [
+  {
+    nickname: 'admin',
+    given_name: 'Admin',
+    family_name: 'User',
+    privilege: 'admin',
+    resource: '*',
+    scope: '*',
+    external_id: randomUUID().slice(0, 6),
```

Comment:
Any chance we could use the same algorithm for these as the actual server also uses? 

https://github.com/curveball/a12n-server/blob/main/src/crypto.ts#L29